it/api/OutboundMessagesApiSpec.scala

```diff
@@ -79,30 +96,122 @@ class OutboundMessagesApiSpec extends AnyFreeSpec with GuiceOneServerPerSuite wi
         </HEAHEA>
       </CC044A>
 
-      repo.insert(arrival).futureValue
+      await(repo.insert(arrival))
```

Comment:
`.futureValue` does not use the default await timeout from the `FutureAwaits` helper so these tests were flaky